crates/db_schema/src/impls/comment.rs

```diff
@@ -184,10 +184,6 @@ impl Saveable for CommentSaved {
     comment_saved_form: &CommentSavedForm,
   ) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
-    let comment_saved_form = (
-      comment_saved_form,
-      comment_actions::saved.eq(now().nullable()),
```

Comment:
No need to force this, now that I added a default `Utc::now()` in the derive_new.